person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id

```

Comment:
- [ ]  Kindly generate random numbers for the instance variable `@id` so that each time a person is created, a random number is generated as the id. Hint; `@id = Random.rand(1..1000)`

- [ ] Since id is generated each time a person is created, we do not need to pass it as a parameter to the initialize method, so kindly remove the `id` parameter from the initialize method.